libnmstate/nm/bridge.py

```diff
@@ -152,12 +172,13 @@ def _get_bridge_port_info(port_profile):
     """Report port information."""
 
     port_setting = port_profile.get_setting_bridge_port()
-    return {
+    port_state = {
```

Comment:
Why change this?

Reply:
Vestigial change that evolution itself cut prematurely.

Got rid of it.